### PR TITLE
add db migration

### PR DESCRIPTION
**Pull request by mrandi**
adding missing migration script for lambda #512

### CODE REVIEW

**Comment by mrandi** [comment]
:+1:

File: fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V16__create_lambda_function_created_from_untrusted_location_type.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  help_text,
+  violation_severity,
+  title,
+  last_modified_by,
+  last_modified,
+  created,
+  created_by,
+  version)
+VALUES ('LAMBDA_FUNCTION_CREATED_FROM_UNTRUSTED_LOCATION',
+        'The Lambda function you created/updated is not stored in our trusted S3 bucket.',
```
Comment (harti2006) [inline fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V16__create_lambda_function_created_from_untrusted_location_type.sql]: 'The code that has been used in a Lambda function, is not stored in of the trusted S3 buckets.'

Comment (harti2006) [reply fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V16__create_lambda_function_created_from_untrusted_location_type.sql]: I would actually like to also put a link to some documentation there. But the docs will only be internal :-/

**Comment by mrandi** [comment]
:+1:

**Comment by harti2006** [comment]
:+1:

**Pull request merged**